README.md

```diff
@@ -610,6 +610,148 @@ resources. Once these are created, a client can query for available samples:
 
 #### TODO `GET /Participant/:pid/BiobankSamples
 
+## Organization API
+
+Retrieves metadata about awardees / organizations / sites.
+
+#### `GET /Awardee`
+
+Retrieves a JSON bundle of metadata for all awardees, with nested resources for child 
+organizations and sites within them, representing the full hierarchy of 
+awardees > organizations > sites. No pagination, syncing, or filtering is currently 
+supported on this endpoint.
+
+Example response:
+
+```
+{
+  "entry": [
+    {
+      "fullUrl": "http://localhost:8080/rdr/v1/Awardee/AZ_TUCSON", 
+      "resource": {
+        "displayName": "Arizona", 
+        "id": "AZ_TUCSON", 
+        "organizations": [
+          {
+            "displayName": "Banner Health", 
+            "id": "AZ_TUCSON_BANNER_HEALTH", 
+            "sites": [
+              {
+                "address1": "1234 Main Street", 
```

Comment:
Might use the FHIR [Address](http://hl7.org/fhir/datatypes.html#address) datatype for the street address piece. As in:

```json
{
  "address": {
    "line": ["1234 Main St", "Suite 400"],
    "city": "Tucson",
    "state": "AZ" -- this is not in the payload now; intentional?,
    "postalCode": "55555"
  }
}
```

Reply:
Done. (And added state, thanks for noticing I was missing that!)